runtime/lua/nvim/health.lua

```diff
@@ -54,9 +54,16 @@ local function check_config()
   health.start('Configuration')
   local ok = true
 
-  local vimrc = (
-    empty(vim.env.MYVIMRC) and vim.fn.stdpath('config') .. '/init.vim' or vim.env.MYVIMRC
-  )
+  local init_lua_path = vim.fn.stdpath('config') .. '/init.lua'
+  local init_vim_path = vim.fn.stdpath('config') .. '/init.vim'
+  if filereadable(init_lua_path) and filereadable(init_vim_path) then
+    health.warn('You can have either Vimscript ("init.vim") or Lua ("init.lua"), but not both.')
```

Comment:
Are you sure this is needed? There's already a builtin check which shows at startup: https://github.com/neovim/neovim/blob/5ea5812714bcbeb6a5438c80d410984ad4fbb922/src/nvim/main.c#L2006

Reply:
It is not needed, have removed